### PR TITLE
onnx softmaxoutput fix

### PR DESCRIPTION
**Pull request by Roshrini**
## Description ##
Fix for the failing tutorial mentioned here
https://github.com/apache/incubator-mxnet/pull/13099
@vandanavk 

### Essentials ###
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by ankkhedia** [comment]
@mxnet-label-bot[pr-awaiting-review, ONNX]

**Comment by anirudhacharya** [comment]
@Roshrini can you give more context why this change is needed. The link you provided just says that there is a bug. What is the bug, how was it introduced, and how is this fixing it?

@zhreshold for review.

**Comment by stu1130** [comment]
@mxnet-label-bot update [pr-work-in-progress]

**Comment by vandanavk** [comment]
@mxnet-label-bot add [pr-awaiting-testing]

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py [hunk 1]
```diff
@@ -681,7 +681,7 @@ def convert_softmax_output(node, **kwargs):
     """Map MXNet's SoftmaxOutput operator attributes to onnx's Softmax operator
     and return the created node.
     """
-    name, _, _ = get_inputs(node, kwargs)
+    name = node["name"]
```
Comment (zhreshold) [inline python/mxnet/contrib/onnx/mx2onnx/_op_translations.py]: The change LGTM, can you add a test case to guard this against happening again?

**Comment by Roshrini** [comment]
@sandeep-krishnamurthy @zhreshold Added test case for the operator

**Pull request merged**